aux_scripts/q_common_modules.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Modules differ across versions, so to produce a time series
+# spanning very old and very new Quarkus, we need to pick
+# just those modules that exist in both. Mind this is a dumb
+# script that doesn't take into account renaming.
+
+QTAGS=(
+"2.13.3.Final" \
+"2.16.12.Final" \
+"3.2.12.Final" \
+"3.8.4" \
+"3.9.5" \
+"3.10.0" \
+"main"
+)
+
+for QTAG in "${QTAGS[@]}"; do
+  curl https://raw.githubusercontent.com/quarkusio/quarkus/main/.github/native-tests.json \
```

Comment:
Shouldn't this fetch the native-tests.json file of the corresponding `QTAG` instead of `main`?

Reply:
:scream:  Yes. Thx!